src/runaqa.ts

```diff
@@ -96,7 +102,11 @@ export async function runaqaTest(
 }
 
 function getTestJdkHome(version: string, jdksource: string): string {
+  // Try JAVA_HOME first and then fall back to GITHUB actions default location
   let javaHome = process.env[`JAVA_HOME_${version}_X64`] as string
+  if (javaHome === undefined) {
+    javaHome = process.env['JAVA_HOME'] as string
```

Comment:
If javaHome is undefined and jdksource === 'github-hosted' the test should stop with error that target JDK is not available.

Reply:
done